.github/workflows/build-devcontainer-image.yml

```diff
@@ -0,0 +1,49 @@
+name: Build Devcontainer image
+on:
+  workflow_dispatch:
+    # no content, allows manual triggering
+
+  schedule:
+    # 5:30 pm every Sunday (UTC)
+    # to pick up any important bug fixes etc in base image
+    - cron: '30 17 * * 7'
```

Comment:
I thought day-of-week was 0-6 ... 

https://en.wikipedia.org/wiki/Cron says 7 is supported on _some_ systems. Maybe use `0` (zero)?

Reply:
The Github actions documentation said to use crontab.guru which gave me: https://crontab.guru/#3_17_*_*_7